specs/lifespan.rst

```diff
@@ -0,0 +1,55 @@
+=================
+Lifespan Protocol
+=================
+
+**Version**: 2.1 (2018-07-01)
```

Comment:
I'd give this its own version. The point of protocols is to allow separate versioning rather than moving them all in lockstep.

Reply:
Done.